demos/cms/cms_denc.c

```diff
@@ -57,7 +57,7 @@ int main(int argc, char **argv)
 
     dout = BIO_new_file("smencr.out", "wb");
 
-    if (!in)
+    if (!in || !dout)
```

Comment:
Our coding style would instead suggest:
```suggestion
    if (in == NULL || dout == NULL)
```

Reply:
Thank you. I have submitted a new commit to fix the coding style.